### PR TITLE
When querying a complete table with pagination, limit is always the set

### PR DESCRIPTION
**Pull request by pypingou**
Only the offset changes.

Fixes https://github.com/fedora-infra/anitya/issues/133

### CODE REVIEW

File: anitya/lib/model.py [hunk 1]
```diff
@@ -399,7 +399,7 @@ def all(cls, session, page=None, count=False):
                 page = None
 
         if page:
-            limit = page * 50
+            limit = 50
             offset = (page - 1) * 50
```
Comment (ralphbean) [inline anitya/lib/model.py]: might as well make this ``offset = (page - 1) * limit`` to make it future-proof.

**Comment by ralphbean** [comment]
This needs to be fixed in 4 or 5 other places in the file also.

Since that code is copied and pasted between all those methods, perhaps it can be reorganized into a single function?

**Comment by ralphbean** [comment]
:+1:

**Comment by pypingou** [comment]
Thanks for the review @ralphbean

**Pull request merged**